### ISSUE TITLE
Add attribute: rights

### ISSUE THREAD

**Issue by shanitajohn** [curationexperts/epigaea#89]
- [x] Have a discussion with Tufts about their current practice versus rights practices in Hyrax
- [ ] Write up any additional change requirements in a new ticket

**Comment by shanitajohn** [comment]
Tufts usually adds this URL:  http://dca.tufts.edu/ua/access/rights-pd.html
Continue using a free text field. 
Display link to the site with the rights

**Comment by no-reply** [comment]
Tufts stores a URL to a webpage with the rights. Mike initially indicated this should be a freetext field, and we're going to move forward that way for the moment (using the existing rights fields in Hyrax)

I recommend that we move away from that, though. There's a limited number of these URLs, so a controlled vocabulary is useful here; we also want to hyperlink them, rather than treat them as text in display, so datatypes are called for. We have two options:

```turtle
# store as xsd:anyURI literal with DC Elements 1.1
:aWork dce: rights "http://example.com/rightsStatement"^^xsd:anyURI
```

```turtle
# store as a resource with DCTerms Rights, i.e. as as RightsStatement
:aWork dct: rights <http://example.com/rightsStatement>
```

I'll reserve judgement on which of these is best until we hear back from @mkorcy about controlled vocabs, etc...

**Comment by mkorcy** [comment]
OK discussed this with Tisch and DCA, their preference would be to do this as a controlled vocabulary, DCA has a few different URLs but they all just redirect to the same place, so we'd have  a DCA rights statement, Tisch uses the DCA statement as well as  few different external statements:
```
https://sites.tufts.edu/dca/about-us/research-help/reproductions-and-use/
http://creativecommons.org/licenses/by/4.0/
https://www.elsevier.com/about/our-business/policies/sharing
```

We have a ~30,000 objects that have a string instead of a URL but we'd just normalize them in the migration.

**Comment by no-reply** [comment]
Thanks @mkorcy! Could you get us a full list of the statement URLs in the next couple of days?

I've reassigned this to myself to work out the predicate/vocab details.

**Comment by mkorcy** [comment]
@no-reply I discussed this with DCA and Tisch yesterday.  There are 20 statements that currently get used across ~200k objects so it makes sense for it to be a controlled vocabulary.  However, there are cases when the value is not going to be a URL, I had thought they could just push these to URLs but they really want it to be the text value in some cases.  I've taken the known the values and put them in a yml file, I think this is what you'll need for this from me.
[licenses.txt](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/56e182fb85a052130057232c/125516c66e82c728ace21e0d46e49d99268c90ac9cabf301b44fff9e2c4533317346f6722180b329eb1428045f5b1ff9415a5e4cffab8c2ea4e22e6c981e5ab38532022d8c6853beb2901db90bbb24824682b1b4894ec4ec3927f5c6.txt)

**Comment by no-reply** [comment]
@mkorcy Is the above comment the current status? It sounded like there might be some late breaking info on this one.

**Comment by mkorcy** [comment]
@no-reply what Tisch and DCA would like to do is say that rights (edm:rights) is a URI controlled vocabulary based on the file above (with the plain text strings removed from that file), but also use dcterms:rights for a plain text that they could use to refine on the basic info in the URI.  Which, looking at hyrax the basic metadata they're righting what I would think should be Licenses (dcterms:license) into dcterm:rights, and I'm not sure why.

The configuration they want is:
Rights URI:
- Predicate: edm:rights
- UI Element: controlled vocabulary selection
- Source: licenses.txt above with final 3 entries removed
Rights Text:
- Predicate: dcterms:rights
- UI Element: free text
License:
 - Predicate: dcterms:license
 - UI Element: text field

**Comment by mkorcy** [comment]
To avoid the conflict with what Hyrax is already doing, they'd be fine with just creating a local predicate for the free text in `TDRTerms` called `RightsNotes`

**Comment by no-reply** [comment]
In check in this morning we discussed the introducing a property using DC Elements 1.1 (rather than DC Terms):

```ruby
property :rights_note, predicate: RDF::Vocab::DC11.rights
```

That gives us:

Rights:

  - Predicate: edm:rights
  - UI Element: controlled vocabulary selection
  - Source: licenses.txt above with final 3 entries removed

License:

   - Predicate: dcterms:rights
   - UI Element: free text

Rights Note:

  - Predicate: dce:rights
  - UI Element: free text


In short, this is base Hyrax with a custom controlled vocabulary and the addition of a single free text property.

@mkorcy: If the above looks correct to you, we can start the technical work on the ticket.

**Comment by mkorcy** [comment]
@no-reply  Looks good to me.
